Assets/Project/Scripts/Modules/StartUpScene/StartUpDirector.cs

```diff
@@ -33,34 +30,21 @@ private IEnumerator Start()
             Debug.Log("PROD");
             #endif
 
-            // Network Initialization
-            // TODO : ユーザー情報の初期化（IDの取得OR発行など）
-            NetworkService.Execute(new HelloWorldRequest(), (data) => {
-                Debug.Log(data as string);
-            });
-
-            // AAS Initialize
-            AddressableAssetManager.Initialize().Completed += OnAASInitializeCompleted;
+            // AASを初期化
+            // => MenuSelectSceneを読み込み
+            // => 開始ボタンをアクティブにする
+            AddressableAssetManager.Initialize()
+                .ToObservable()
+                .Subscribe(_ => {
+                    AddressableAssetManager.LoadScene(Constants.SceneName.MENU_SELECT_SCENE, LoadSceneMode.Additive)
+                        .ToObservable()
+                        .Subscribe(__ => _startButton.SetActive(true));
+                });
```

Comment:
UniRx では基本的にメモリリーク対策で購読解除のコードを書く必要があるのだけど（以下のコードは `this` が破棄されたタイミングに購読を解除する）、UniTask から UniRx に変換（ `.ToObservable` ）した場合には購読解除しなくて良い感じ？（必ず `OnCompleted` 流れるとか？）
```suggestion
                .Subscribe(_ => {
                    AddressableAssetManager.LoadScene(Constants.SceneName.MENU_SELECT_SCENE, LoadSceneMode.Additive)
                        .ToObservable()
                        .Subscribe(__ => _startButton.SetActive(true));
                })
                .AddTo(this);
```

Reply:
あんまりよくわかっていないけど、 `UniTask` の解除は [`WithCancellation`](https://github.com/Cysharp/UniTask#cancellation-and-exception-handling)かな

Reply:
Observable に変換してるよね。そしたら AddTo が必要に思える。"UniRx 購読 メモリリーク" とかって調べるとわかるよ

Reply:
あ、ここは手続きにしようかなって思いました
イベント待ち→イベント待ちになってるのと
`_startButton` 本当は下のデータベースの初期化も終わってからアクティブする方が妥当だから
一旦この変更無視してほしい

Reply:
んん、どういうこと？とりあえず、 `Subscribe` が使うならメモリリークの話が出てくるからそこだけ理解してもらえれば！

Reply:
メモリリークの話はわかった！
僕が言ったことは新しいコミット見てもらえればわかると思います

Reply:
UniTask だと cancellation というのがあるのか🤔 まだレビューは続く・・・ということで

Reply:
`await AddressableAssetManager.Initialize();` みたいに `UniRx` ではなく `UniTask` を使ったのは良いと思う！Slack でも言ったけど、一回で終わる非同期処理などは `UniTask` が相性良いみたい！（→ [Slack](https://lito-apps.slack.com/archives/C01E5GTPRV3/p1607515173019400)）